lib/matplotlib/axes/_base.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import (absolute_import, division, print_function,
+﻿from __future__ import (absolute_import, division, print_function,
```

Comment:
It's probably something that your editor is doing silently for you to be helpful.  How you turn that off would be editor specific.